csharp/TheBuildingClass/UnitTestProject1/Building.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
about naming conventions, generally the structure for tests is something like:
* \<projName\>
    * \<folder\>
        * \<className\>
* \<projName\>.Unit.Tests
   * \<folder\>
        * \<className\>Tests

this way it's always recognisable if the class is the test class or the actual implementation